components/ThemeList.jsx

```diff
@@ -85,6 +89,9 @@ class ThemeList extends React.Component {
         const addLayersTitle = LocaleUtils.tr("themeswitcher.addlayerstotheme");
         const addTitle = LocaleUtils.tr("themeswitcher.addtotheme");
         const openTabTitle = LocaleUtils.tr("themeswitcher.openintab");
+        const changeDefaultUrlTitle = LocaleUtils.tr("themeswitcher.changedefaulttheme");
+        const username = ConfigUtils.getConfigProp("username");
+
```

Comment:
Please sort props alphabetically (resp. configure eslint in your editor).

Reply:
Sorry for that !